app.rb

```diff
@@ -5,5 +5,29 @@ class Chitter < Sinatra::Base
     'Test page'
   end
```

Comment:
It's good to remove any extraneous code from your implementation before making a Pull Request - there'll be lots of testing and exploration as you work on something, but it tidies it up to remove anything that is no longer needed for the program to meet its specification.